sentry_sdk/integrations/__init__.py

```diff
@@ -144,8 +144,14 @@ def setup_integrations(
                     logger.debug(
                         "Did not enable default integration %s: %s", identifier, e
                     )
-
-                _installed_integrations.add(identifier)
+                else:
+                    _installed_integrations.add(identifier)
```

Comment:
We're changing the semantics of the `_installed_integrations` global var from "integrations we've tried installing" to "integrations we've successfully installed" here. This could potentially be an issue with multiple clients since they might try reinstalling the failed integrations again because they're now not in `_installed_integrations` anymore.

I'd suggest we create a new local variable to keep track of what was actually installed and use that for the check on lines 150-154, while keeping `_installed_integrations` as is.

(Maybe we can find a better name for `_installed_integrations` though. `_processed_integrations`?)

Reply:
Ok. I renamed the global variable to _install_attempted_integrations since I think it's more descriptive than _processed_integrations. Although, _processed_integrations might be the better name since it's less verbose, what is your preference?

Reply:
I'd prefer the shorter name but it's fine either way.